src/main/resources/migrations.xml

```diff
@@ -5778,4 +5778,44 @@
 		/>
 	</changeSet>
 
+	<changeSet id="add-entityAvatars-table" author="fdelcastillo">
+		<createTable tableName="entityAvatars">
+			<!-- The next two columns combined act as a generic foreign key reference to a (table name, uuid)-tuple -->
+			<column name="relatedObjectType" type="varchar(255)">
+				<constraints nullable="false" />
+			</column>
+			<column name="relatedObjectUuid" type="${uuid_type}">
+				<constraints nullable="false" />
+			</column>
+			<column name="attachmentUuid" type="${uuid_type}">
+				<constraints nullable="false" />
+			</column>
+			<column name="applyCrop" type="boolean">
+				<constraints nullable="false" />
+			</column>
+			<column name="cropLeft" type="number">
+				<constraints nullable="false" />
+			</column>
+			<column name="cropTop" type="number">
+				<constraints nullable="false" />
+			</column>
+			<column name="cropWidth" type="number">
+				<constraints nullable="false" />
+			</column>
+			<column name="cropHeight" type="number">
+				<constraints nullable="false" />
+			</column>
+		</createTable>
+		<addUniqueConstraint
+				tableName="entityAvatars"
+				columnNames="relatedObjectUuid"
+				constraintName="uniqueEntityAvatars"
+		/>
+		<addForeignKeyConstraint
+				baseColumnNames="attachmentUuid" baseTableName="entityAvatars"
+				constraintName="fk_entityAvatars_attachment" onDelete="CASCADE" onUpdate="NO ACTION"
+				referencedColumnNames="uuid" referencedTableName="attachments"
+		/>
+	</changeSet>
 </databaseChangeLog>
+
```

Comment:
Remove this trailing empty line, and keep the empty line _before_ the final closing tag in this file.